R/generate_wrappers.R

```diff
@@ -251,16 +253,19 @@ generateTripleComboMatrix <- function(cell_lines, drugs, save = TRUE) {
     c(0, concentration)
   )
   df_2 <- changeColNames(df_2, drugs, "_2")
-  df_layout_2 <- merge(df_layout, df_2, allow.cartesian = TRUE)
+  df_layout_2 <- df_layout[df_2, on = intersect(names(df_layout), names(df_2)),
+                           allow.cartesian = TRUE]
   
   df_3 <- prepareData(
     cell_lines[cell_lines$clid %in% df_layout$clid, ], 
     drugs[10, ],
     c(0, .1, 1)
   )
   df_3 <- changeColNames(df_3, drugs, "_3")
-  df_layout_3 <- merge(merge(df_layout, df_2, allow.cartesian = TRUE),
-                       df_3, allow.cartesian = TRUE)
+  df_layout_3 <- df_layout[df_2, on = intersect(names(df_layout), names(df_2)),
+                           allow.cartesian = TRUE][df_3, on = intersect(names(df_layout),
+                                                                        names(df_3)),
+                                                   allow.cartesian = TRUE]
```

Comment:
```suggestion
  df_layout_3 <- df_layout[df_2, on = intersect(names(df_layout), names(df_2)), allow.cartesian = TRUE
                                      ][df_3, on = intersect(names(df_layout), names(df_3)), allow.cartesian = TRUE]
```

For better readability